sections/introduction.tex

```diff
@@ -16,14 +24,15 @@ \section{Introduction}\label{sec:introduction}
 % Metadata definition
 We define this other data as metadata, that is data related to the documents in the corpus, but not directly a part of the text.
 In most models, metadata is not used.
-However, there is some previous work that attempts to make full use of the dataset available by incorporating various metadata into topic models.
+However, there is some previous work that attempts to make full use of the dataset available by incorporating various metadata into topic models.\todo[inline]{Giving a few paper sources here would be good to give the sentence more context.}
```

Comment:
```suggestion
However, there is some previous work that attempts to make full use of the dataset available by incorporating various metadata into topic models, such as the Author-Topic model by \citet{author_topic_2012} and the MetaLDA model by \citet{MetaLDA2017}.
```
Kunne det f.eks. være. Vi kan også bruge nogle andre kilder i stedet, hvis vi vil undgå at bruge author-topic/metaLDA for meget.